Say Their Names/Source/Controller/Home/HomeView.swift

```diff
@@ -0,0 +1,160 @@
+//
+//  HomeView.swift
+//  Say Their Names
+//
+//  Created by Joseph A. Wardell on 6/2/20.
+//  Copyright © 2020 Franck-Stephane Ndame Mpouli. All rights reserved.
+//
+
+import UIKit
+
+class HomeView: UIView {
+
+    var customNavBar: UIView!
+    var locationCollectionView: UICollectionView!
+    var peopleCollectionView: UICollectionView!
+    var searchButton: UIButton!
+    var separator : UIView!
+    
+    var hideable : [UIView] {
+        [customNavBar, locationCollectionView, peopleCollectionView, separator]
+    }
+    
+    required init?(coder: NSCoder) { fatalError("This should never be called") }
+    override init(frame: CGRect) {
+        
+        let customNavBar = UIView()
+        customNavBar.backgroundColor = .black
+        self.customNavBar = customNavBar
+        
+        let locationLayout = UICollectionViewFlowLayout()
+        locationLayout.scrollDirection = .horizontal
+        locationLayout.sectionInsetReference = .fromContentInset
+        locationLayout.sectionInset = Self.LocationsSectionInsets
+        let locationCollectionView = UICollectionView(frame: .zero, collectionViewLayout:locationLayout)
+        locationCollectionView.tag = 0
+        locationCollectionView.contentInsetAdjustmentBehavior = .always
+        self.locationCollectionView = locationCollectionView
+        
+        let peopleLayout = UICollectionViewFlowLayout()
+        peopleLayout.scrollDirection = .vertical
+        peopleLayout.sectionInset = Self.PeopleSectionInsets
+        let peopleCollectionView = UICollectionView(frame: .zero, collectionViewLayout:peopleLayout)
+        peopleCollectionView.tag = 1
+        peopleCollectionView.contentInsetAdjustmentBehavior = .always
+        self.peopleCollectionView = peopleCollectionView
+        
+        let separator = UIView()
+        separator.backgroundColor = .systemGray6
+
+        self.separator = separator
+        
+        let searchButton = UIButton(type: .custom)
+        let searchImage = UIImage(named: "Simple Search Icon")!
+        searchButton.setImage(searchImage, for: .normal)
+        self.searchButton = searchButton
+        
+        super.init(frame: frame)
+
+        backgroundColor = .black
+    }
+    
+    override func didMoveToSuperview() {
+        super.didMoveToSuperview()
+        
+        createLayout()
+        
+    }
+    
+    private var hasLayedOutSubviews = false
+    private func createLayout() {
+        guard !hasLayedOutSubviews else { return }
```

Comment:
`updateConstraints()` override is a more appropriate place for setting up constraints. the end result would be pretty much the same :)

Reply:
Agreed.  Done.  It also let me find and fix a weird layout bug when moving from portrait to landscape. Thanks.

Reply:
🙌